lnwire/message.go

```diff
@@ -63,6 +63,25 @@ const (
 	MsgKickoffSig                          = 777
 )
 
+// IsChannelUpdate is a filter function that discerns channel update messages
+// from the other messages in the Lightning Network Protocol.
+func (t MessageType) IsChannelUpdate() bool {
```

Comment:
What about commit sig and revoke?

Reply:
Sig and revoke are not updates. They roll the updates up into new state advancements etc, but they are not themselves updates.